src/bond/james.cljc

```diff
@@ -66,6 +66,34 @@
                       (vec))
      ~@body))
 
+(defn- var-arg? [arglist]
+  (some #{'&} arglist))
+
+(defn- args-match? [args {:keys [arglists name ns]}]
+  (or (some var-arg? arglists)
+      (some #(= (count args) (count %)) arglists)))
```

Comment:
~ How hard would it be to tweak this to look for a matching minimum number of args in the varargs case? When stubbing `(fn [a b & c] ...)` we should be passing >= 2 args.